### ISSUE TITLE
Using custom templates in Windows, folder separation is not changed (from \ to /)

### ISSUE THREAD

**Issue by kuworking** [kevin940726/remark-codesandbox#12]
I rewrite the question since this didn't have sense

I am using a custom template by (as said in the docs) downloading files with `export ZIP`

The gatsby config is like this

```
          {
            resolve: 'gatsby-remark-inline-codesandbox',
            options: {
              mode: 'meta',
              customTemplates: {
                react_local: {
                  extends: 'file:./src/components/mdx/codesandbox-react',
                  entry: 'src/App.js',
                },
              },
            },
          },
```

The problem that I have is that I'm using Windows 10, and folders are `codesandbox-react\src\index.js`

So when I'm using this, I see that codesandbox expects the `/` linux separation

![image](https://user-images.githubusercontent.com/4474004/94058396-3c569e80-fde1-11ea-916b-4b656c40c731.png)

Is there any workaround I could use here?

--------------

OLD QUESTION

I was trying this but I've noticed now that it is not working

```
          {
            resolve: 'gatsby-remark-inline-codesandbox',
            options: {
              mode: 'meta',
              customTemplates: {
                react_local: {
                  extends: 'file:./src/components/mdx/codesandbox-react',
                  entry: 'src/App.js',
                },
              },
            },
          },
```

The entry creates a unique `App.js` file, and without the entry it creates a unique `index.js` file, instead of using the typical template with both files

~~Maybe this is related to the template, but the thing is that I cannot remember from where I did download the template, which is a json file (from here https://github.com/kevin940726/remark-codesandbox#markdown-syntax)~~

~~So ... from where should I download these templates? 😅~~

EDIT: Yes, from here
https://codesandbox.io/api/v1/sandboxes/react-new

So with this downloaded template, the above code should work or I've made a mistake?

**Comment by kevin940726** [comment]
Thanks for reporting! This is indeed a bug in Windows! I'll try to fix it as soon as possible :)

**Comment by kevin940726** [comment]
Published `remark-codesandbox` v0.8.2. Please give it a try!

If you're using `gatsby-remark-inline-codesandbox`, remove it and try re-installing it again. The package manager should be able to grab the latest release.
